src/hotspot/cpu/aarch64/aarch64_neon.ad

```diff
@@ -3454,3 +3454,166 @@ instruct alltrue_in_mask16B(iRegINoSp dst, vecX src1, vecX src2, vecX tmp, rFlag
   %}
   ins_pipe(pipe_slow);
 %}
+
+// --------------------------------- ABS --------------------------------------
+
+instruct vabs8B(vecD dst, vecD src)
+%{
+  predicate(n->as_Vector()->length() == 8);
```

Comment:
This is different from original code.

Reply:
For integer absolute (vector), the accepted arrangements are `T8B, T16B, T4H, T8H, T2S, T4S, T2D`.
ARM compiler armasm user guide reference: https://developer.arm.com/documentation/dui0801/h/A64-SIMD-Vector-Instructions/ABS--vector-?lang=en
I think the original code `n->as_Vector()->length() == 4 ||` is not right for basic type Byte. So I delete it, I am sorry if I miss something.

Reply:
We load 4B with loadV4 but handle 4B types with T8B instructions. And current min_vector_size for byte type is 4, so it's possible for vectorizer to generate 4B vector nodes.

Reply:
Oh, I see. Thanks a lot for clarifying this, I should have taken a deep look.
Really sorry for producing a BUG here, I will fix it.